src/features/bookings/pages/BookingDetails.tsx

```diff
@@ -121,6 +121,7 @@ export function BookingDetails() {
   const shouldDisplayItineraryButton =
     canOpenItinerary && (properties.isEvent || (properties.isPhysical && !properties.isDigital))
   const activationCodeFeatureEnabled = appSettings && appSettings.autoActivateDigitalBookings
+  const proDisableEventsQrcode = appSettings && appSettings.proDisableEventsQrcode
```

Comment:
est-ce que ça serait suffisant ?
```suggestion
  const proDisableEventsQrcode = appSettings?.proDisableEventsQrcode
```

Reply:
Yes c'est ok comme ça